### PR TITLE
Supply completions for negative regex categories as well.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
View with whitespace off.  this is a trivial change.

### CODE REVIEW

**Comment by mavasani** [comment]
@CyrusNajmabadi Is this fixing a specific issue or just something that you observed?

**Review by mavasani — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
>  or just something that you observed?

An observation :)

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_Regex.vb [hunk 1]
```diff
@@ -271,6 +271,30 @@ class c
...
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestNegativeCategory(showCompletionInArgumentLists As Boolean) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                <Document><![CDATA[
+using System.Text.RegularExpressions;
+class c
+{
+    void goo()
+    {
+        var r = new Regex(@"\P$$");
```
Comment (Youssef1313) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_Regex.vb:283]: worth a test with target-typed expression?

Comment (CyrusNajmabadi) [reply src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_Regex.vb:283]: The code change doesn't involve that scenario, so I don't think that's worthwhile to add a part of this :-)

**Pull request merged**